examples/generated/test/sketchJs/style/TextStyleConditional.js

```diff
@@ -16,7 +16,7 @@ export default class TextStyleConditional extends React.Component {
     }
     return (
       <View style={[ styles.view, {} ]}>
-        <Text style={[ styles.text, { font: Text$textStyle } ]}>
+        <Text style={[ styles.text, { ...Text$textStyle } ]}>
```

Comment:
actually I'm wondering if `style={[ styles.text, Text$textStyle ]}` is enough. It should be, right?

Reply:
Yeah. Let's do that as an optimization though, cause I don't have time right now :P